ExtraHop - Add incorrectly deprecated playbook back into ExtraHop Pack (#9226)
demisto/content: Packs/ExtraHop/ReleaseNotes/1_0_3.md

#### Playbooks
##### ExtraHop - Ticket Tracking

> - Re-enabled the playbook as it was deprecated.

Reviewer: ```suggestion
- Playbook was un-deprecated and is now available.
```
